### PR TITLE
Create controller

### PR DESCRIPTION
**Pull request by Honcha-renko**
Create controller.
Added methods and views for operations create, delete and read.

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+    @task = Task.find(params[:id])
+  end
+
+  def create
+    @task = Task.create(task_params)
+    redirect_to @task
+  end
+
+  def update
+    @task = Task.find(params[:id])
```
Comment (aantsyferova) [inline app/controllers/tasks_controller.rb]: replace @task = Task.find(params[:id]) with before_action callback

File: app/views/tasks/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <th></th>
+  </tr>
+
+  <% @tasks.each do |task| %>
+<!--<%# if task.done %>//-->another style
+    <tr>
+      <td><%= task.title %></td>
+      <td><%= task.tag %></td>
+      <td><%= task.done %></td>
+      <td><%=task.expire_on.strftime("%F %T") unless task.expire_on.nil?%></td>
```
Comment (aantsyferova) [inline app/views/tasks/index.html.erb]: whitespaces missing

File: app/views/tasks/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      <td><%= link_to 'Show', task_path(task) %></td>
+<!--      <td><%#= link_to 'Edit', task, :method => :put %></td>-->
+      <td><%= link_to "Delete", task, :method => :delete%></td>
```
Comment (aantsyferova) [inline app/views/tasks/index.html.erb]: whitespace missing at the end

File: app/views/tasks/index.html.erb (hunk 2 above)
Comment (Honcha-renko) [reply app/views/tasks/index.html.erb]: Do you mean after nil? ?

**Pull request merged**